packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -44,11 +44,23 @@ const getSSMValue = async (key: string | undefined): Promise<string> => {
   // the security group's inbound rule for port 443 is not defined,
   // the SSM client waits for the entire lambda execution time and times out.
   // If the parameter is not retrieved within 10 seconds, throw an error.
-  const data = await Promise.race([secretsClient.send(parameterCommand), wait10SecondsAndThrowError()]);
-  if ((data?.$metadata?.httpStatusCode && data?.$metadata?.httpStatusCode >= 400) || !data?.Parameter?.Value) {
+  const data = await Promise.race([secretsClient.send(parameterCommand), wait10Seconds()]);
+
+  // If string is returned, throw error.
+  if (
+    (typeof data === 'string' || data instanceof String) &&
+    data === WAIT_COMPLETE
+  ) {
+    console.log('Unable to retrieve secret for database connection from SSM. If your database is in VPC, verify that you have VPC endpoints for SSM defined and the security group\'s inbound rule for port 443 is defined.');
```

Comment:
```suggestion
    console.log('Unable to retrieve secret for database connection from SSM. If your database is in a VPC, verify that you have VPC endpoints for SSM defined and the security group\'s inbound rule for port 443 is defined.');
```

Reply:
Also, it's actually like 4 or 5 endpoints right? We should add a TODO to add a doc link for that so customers can go to setup instructions.

Reply:
Since we are using local VPC endpoint URLs, this inbound rule shouldn't be required now. This is meant to be more generic message if we are unable to connect to the SSM for any reason, we have a backlog item to return more meaningful message.